ocaml/sdk-gen/component-test/README.md

```diff
@@ -0,0 +1,60 @@
+## How Component Testing Works
+
+#### jsonrpc-server
+The jsonrpc-server is a mock HTTP server created to emulate the XAPI (eXtensible Application Programming Interface) environment. It operates by executing the method specified within incoming requests and subsequently providing a response that includes the outcome of that execution. To facilitate comprehensive testing, the server extracts test data from JSON files located in the spec/ directory. To enhance the organization and scalability of test cases, especially when simulating various API versions for the purpose of ensuring backwards compatibility, the following structure is recommended:
```

Comment:
Xapi doesn't stand for that, has this been generated using AI?

Reply:
> Xapi doesn't stand for that, has this been generated using AI?
Yes, I used AI rewrite my words, I will update this.

Reply:
I also favour one sentence per line, but it doesn't bother much when reviewing as the wrap is done by github

Reply:
Where else was AI used in this PR or other PRs?

Reply:
> Where else was AI used in this PR or other PRs?

To get more accurate translations, all the documents are rewritten by AI.

Reply:
Generating documents is worse than generating code. At least the compiler can help check the generated code for the safety, while no tools can do this against the generated documents for the same purpose.

Reply:
OK, I see,  I will keep using my initial documents with my words.